### ISSUE TITLE
Moment.js

### ISSUE THREAD

**Issue by blahah** [hubgit/dates#1]
Just checking - I assume you do - but do you know about [Moment.js](momentjs.com)? Because this is why it exists.

**Comment by hubgit** [comment]
Yes, Moment is great! I use it in several projects, but in this case I just wanted a scratchpad for trying out (and remembering) various date-related functions and interfaces.

**Comment by blahah** [comment]
:)
